applications/models.py

```diff
@@ -11,6 +11,7 @@
 
 from app import utils
 from user.models import User
+from .validators import validate_file_extension
```

Comment:
No relative path pls
```suggestion
from applications.validators import validate_file_extension
```